Source/Core/DiscIO/VolumeDirectory.cpp

```diff
@@ -497,4 +505,20 @@ static u32 ComputeNameSize(const File::FSTEntry& parent_entry)
   return name_size;
 }
 
+static char ASCIILowercaseChar(char c)
+{
+  if (c >= 'A' && c <= 'Z')
+    c += 0x20;
+  return c;
+}
+
+static std::string ASCIILowercaseString(const std::string& str)
+{
+  std::string copy = str;
+  // We can't use tolower, because its behavior depends on the locale,
+  // and we want to be deterministic
+  std::transform(copy.begin(), copy.end(), copy.begin(), ASCIILowercaseChar);
```

Comment:
If you include `<locale>` you can do std::tolower(value, std::locale::classic());